scripts/setupLift.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+mkdir -p ~/.m2
+cat <<EOF > ~/.m2/settings.xml
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                      https://maven.apache.org/xsd/settings-1.0.0.xsd">
+  <localRepository>/home/lift/.m2/repository</localRepository>
```

Comment:
Avoids assuming `~` is `/home/lift` (untested).


```suggestion
M2_HOME=~/.m2
mkdir -p ${M2_HOME}
cat <<EOF > ${M2_HOME}/settings.xml
<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
                      https://maven.apache.org/xsd/settings-1.0.0.xsd">
  <localRepository>${M2_HOME}/repository</localRepository>
```

Reply:
Tested and working fine. Thanks @k-wall !!

Reply:
That confused me, when I was testing M2_HOME didn't seem to be set. If it works great!